.travis.yml

```diff
@@ -136,16 +136,12 @@ matrix:
         - COVERAGE=Y
         - NO_HSTORE=Y
         - CREATE_PLPGSQL=Y
-    - jdk: # JDK9 EA Azul Zulu OpenJDK
+    - jdk: oraclejdk9
       sudo: required
```

Comment:
Try using `sudo: false`, it should be faster since it use container based infrastructure.

Reply:
+1. does `sudo: false` work here?